### PR TITLE
Added social links 

### PR DESCRIPTION
**Pull request by shivamsinghydv**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6229"></a>

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
kindly check the position of brackets rest all is great

File: data/shivamsinghydv.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      "url": "https://twitter.com/SaraJaoude",
+      "icon": "FaTwitter"
+    },{
+      "name": "Github",
+      "url": "https://github.com/shivamsinghydv",
+      "icon": "FaGithub"
+    },{
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/shivamsinghydv/",
+      "icon": "FaLinkedIn"}
```
Comment (Sindhuinti) [inline data/shivamsinghydv.json:17]: typo in icon name please change it to `FaLinkedin`

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/shivamsinghydv.json (hunk 1 above)
Comment (shivamsinghydv) [reply data/shivamsinghydv.json]: Thanks

**Comment by SaraJaoude** [comment]
You have conflicts in your file, please resolve. This is usually because your fork is not up to date.  

You may wish to have a look at the following Docs and video: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Comment by shivamsinghydv** [comment]
Hey There, I've updated my fork still the conflict isn't resolved. I need guidance.

**Comment by SaraJaoude** [comment]
It looks like you closed your Pull Request.  I shared the editing guide above which has a step by step guide (and video) on how to make changes to your profile. 

If you are still having trouble then I would suggest asking in the help channel in the EddieHub Discord.

**Pull request closed without merging**